specific/cmo_sale/models/sale.py

```diff
@@ -37,6 +37,7 @@ class SaleOrder(models.Model):
     project_related_id = fields.Many2one(
         'project.project',
         string='Project',
+        inverse='_compute_project_related_id',
```

Comment:
Why you have inverse without compute ?
(Normally inverse works together with compute)
I think this inverse='...' should be in product_id and project_number isn't it?
Note: I think you have wrong use of compute field. Please come to me.